cluster-up/cluster/kind-k8s-1.17.0-ipv6/provider.sh

```diff
@@ -24,10 +24,22 @@ function up() {
             mount_disk $node $i
         done
         $DOCKER exec $node bash -c "chmod -R 777 /var/local/kubevirt-storage/local-volume"
+
+        # Create a unique UUID file reference file for each node, that will be mounted in order to support
+        # Migration in Kind providers.
+        $DOCKER exec $node bash -c 'cat /proc/sys/kernel/random/uuid > /kind/product_uuid'
```

Comment:
this is already merged thanks to Daniel in kubevirtci
there is a PR that will bump it to kubevirt and then it will gone from this PR